requirements.txt

```diff
@@ -1,9 +1,10 @@
 numpy>=1.13.3
-pandas>=0.23.0
+pandas>=0.23.0,<0.24.0
 tqdm>=4.19.2
-featuretools>=0.7.0
+featuretools>=0.7.0,<0.12.0
 nltk>=3.4
 tensorflow<=1.14.0
 tensorflow_hub<=0.4.0
 scikit-learn>=0.20.0,<0.21; python_version < '3'
-scikit-learn>=0.20.0; python_version >= '3'
\ No newline at end of file
+scikit-learn>=0.20.0; python_version >= '3'
+python-dateutil==2.8.0
```

Comment:
I think a range would be better, for featuretools I used
`python-dateutil>=2.6.1,<2.8.1`

Reply:
updated to range